Update references from claim->statement and ledger-->append-only log

SteveLasker opened the pull request:
This PR addresses the root `readme.md` to ensure the generic references to Statement flow when used in descriptive sentences. 
This includes
- Capitalizing `S` in `Statement` to refer to the term.
- Use of `append-only log` - should this also be capitalized `Append-only log`?

If approved, a scrub of the remaining pages will be updated.

henkbirkholz commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
 ### eNotary
 
-SCITT is analogous to a digital, or electronic notary service (eNotary), where minimal information is written to the ledger, endorsing the claim. When users notarize legal documents, the notary ledger records the verified identity of the parties, referencing the legal document they are notarizing. The notary ledger doesn't store the legal document, but does have a reference to it. 
+SCITT is analogous to a digital, or electronic notary service (eNotary), where minimal information is written to the append-only log, counter-signing the Statement. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does ha ...
```
```suggestion
SCITT is analogous to a digital or electronic notary service (eNotary), where minimal information is written to the append-only log and an augmented counter-signature over the Statement and corresponding metadata (Receipt) is produced. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does have a reference to it. 
```

henkbirkholz commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
+SCITT is analogous to a digital, or electronic notary service (eNotary), where minimal information is written to the append-only log, counter-signing the Statement. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does ha ...
 
-In SCITT, the ledger will contain pointers to the artifact, which the claims are made, with pointers to any supporting evidence.
+In SCITT, the append-only log will contain pointers to the artifact, which Statements are made, with optional pointers to any supporting evidence.
```
Is that found in specification text today?

SteveLasker replied:
This is just updating terminology of ledger to append-only log
Are you asking if this was copy/pasted from the spec?

SteveLasker replied:
@henkbirkholz, the only difference I've added was the word **optional**.  This refers to detached content. I'll make a separate PR for updating the artifact=feed and evidence = statement body. 
Any objections to merging this change for claim=statement and ledger=append-only log?

OR13 replied:
please merge, its an improvement.

yogeshbdeshpande replied:
Again, we never store Transparent Statements inside log, if I understood correctly. We issue Transparent Statements by attaching Receipts to the Signed Statements!

henkbirkholz commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
 ### Evidence Persistence
 
-A SCITT instance will persist verifiable claims to its ledger. Any optional evidence will be persisted in associated storage. 
+A SCITT instance will persist Transparent Statements to its append-only log. Any optional evidence will be persisted in associated storage. 
```
What would be an example of 'optional evidence'?

SteveLasker replied:
SBOM, VEX, Scan Report, and supporting document that is either in, or referenced by the COSE Envelope

OR13 reviewed (approved):
This PR should be merged immediately, issues should be raised for any threads that were not resolved at the time of merging.

yogeshbdeshpande commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
 ## SCITT Persistence
 
-SCITT is intended to store verifiable claims for the life of the of the SCITT instance. One of the many questions that surfaces is how big will the SCITT ledger get? What kind of data will go on the SCITT ledger that would cause it to grow?
+SCITT is intended to store Transparent Statements for the life of the of the SCITT instance. One of the many questions that surfaces is how big will the SCITT append-only log get? What kind of data will go on the SCITT append-only log that would cause it to grow?
```
```suggestion
SCITT is intended to store Signed Statements for the life of the of the SCITT instance. One of the many questions that surfaces is how big will the SCITT append-only log get? What kind of data will go on the SCITT append-only log that would cause it to grow?
```

yogeshbdeshpande commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
 Detailed information comes in varying formats, from structured to unstructured.
 
-In SCITT, structured data is a claim. A claim is a well-structured statement, made by a verifiable entity that may have supporting evidence.
+In SCITT, structured data is represented as a Statement. A Statement is a well-structured statement, made by a verifiable entity that may have supporting evidence.
```
But it may just be ok, here as the supporting evidence is the signature I suppose!

SteveLasker replied:
The terminology used for "evidence" is another area we should clarify. I kept this PR to focus on claim--> statement.

yogeshbdeshpande commented on README.md:
```diff
@@ -15,30 +15,30 @@ SCITT supports the ongoing verification of goods and services where the authenti
 ### Evidence Persistence
 
-A SCITT instance will persist verifiable claims to its ledger. Any optional evidence will be persisted in associated storage. 
+A SCITT instance will persist Transparent Statements to its append-only log. Any optional evidence will be persisted in associated storage. 
```
```suggestion
A SCITT instance will persist  Signed Statements to its append-only log. Any optional evidence will be persisted in associated storage. 
```

SteveLasker commented:
Thanks for the reviews. I've addressed the feedback, focused on this PR, merged in the refinements from @yogeshbdeshpande on signed vs. transparent. 
Merging, pending additional updates for evidence and such.
